.github/workflows/sbom.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
       id-token: write   # This is required for requesting the JWT
       contents: read    # This is required for actions/checkout
     needs: changedfiles
-    if: contains(github.ref, 'release-') ||  contains(github.head_ref, 'release-') || contains(github.ref, 'master') || needs.changedfiles.outputs.go || needs.changedfiles.outputs.npm || needs.changedfiles.outputs.ci || needs.changedfiles.outputs.docker || needs.changedfiles.outputs.github
+    if: contains(github.base_ref, 'release-') ||  contains(github.base_ref, 'release-') || contains(github.base_ref, 'master') || needs.changedfiles.outputs.go || needs.changedfiles.outputs.npm || needs.changedfiles.outputs.ci || needs.changedfiles.outputs.docker || needs.changedfiles.outputs.github
```

Comment:
The first condition is duplicated, I asume one needs to remain `head_ref`

Reply:
good catch, one of them can be removed.